### PR TITLE
Logo Functionality Changes

### LINKED ISSUE #705: Fix DLP Logo
**Describe the bug**
Hover functionality on the DLP logo on the very top left in the Nav Bar doesn't work since the nextjs Image component doesn't really work too well with Bootstrap components. Also positioning of the logo is implemented in a hacky way (sorry had to implement something quick to get the nextjs upgrade through).

**To Reproduce**
Hover over the DLP logo.

**Expected behavior**
The logo should change color in the same way as the link next to it.

**Screenshots**
image

### PR DESCRIPTION
**Pull request by prishavall**
# Logo Functionality Changes

**What user problem are we solving?**
Previously, hover functionality on the DLP logo on the very top left in the Nav Bar doesn't work since the nextjs Image component doesn't really work too well with Bootstrap components. 
**What solution does this PR provide?**
The logo should changes color in the same way as the link next to it as you hover.
**Testing Methodology**
How did you test your changes and verify that existing 
functionality is not broken
Ran local host and hovered/clicked to see functionality
**Any other considerations** [New Recording - 8/16/2023, 4:17:22 PM](https://vimeo.com/855150485/05efc37af5)

### CODE REVIEW

**Comment by karkir0003** [comment]
@farisdurrani

**Comment by karkir0003** [comment]
@prishavall can u post a brief video/screen recording?

**Review by farisdurrani — APPROVED** [review]

**Comment by farisdurrani** [comment]
Removed some commented code. @prishavall please resolve the conflicts and you can merge it in.

Note: the changes in .aws/task-definition.json are purely Prettier

**Comment by farisdurrani** [comment]
Good job

**Comment by farisdurrani** [comment]
Hey @prishavall can you solve the conflicts and merge this?

File: frontend/src/common/components/NavBarMain.tsx [hunk 1]
```diff
@@ -87,6 +87,13 @@ const NavbarMain = () => {
                   noWrap
                   className="d-flex align-items-center logo-title"
                   sx={{
+
+                  mr: 2,
+                   display: { xs: "none", md: "flex" },
+                  //fontFamily:
+                  //color: inherit
+                   textDecoration: "none",
+=======
```
Comment (karkir0003) [inline frontend/src/common/components/NavBarMain.tsx]: remove this @prishavall

File: frontend/src/common/components/NavBarMain.tsx [hunk 2]
```diff
@@ -168,7 +175,11 @@ const NavbarMain = () => {
                         </Link>
                       </MenuItem>
                       <MenuItem divider>
+
+                        <Link href="/learn-mod" id="basic-nav-dropdown">
+=======
```
Comment (karkir0003) [inline frontend/src/common/components/NavBarMain.tsx]: remove this @prishavall

**Review by karkir0003 — CHANGES_REQUESTED** [review]
@prishavall minor fixes. see my comments

File: frontend/src/common/components/NavBarMain.tsx [hunk 3]
```diff
@@ -168,7 +173,10 @@ const NavbarMain = () => {
                         </Link>
                       </MenuItem>
                       <MenuItem divider>
+
+                        <Link href="/learn-mod" id="basic-nav-dropdown">
```
Comment (karkir0003) [inline frontend/src/common/components/NavBarMain.tsx]: You mean `<Link href="/learn-mod" id="basic-nav-dropdown"/>`

**Review by karkir0003 — CHANGES_REQUESTED** [review]
see comment on closing tag

**Review by karkir0003 — APPROVED** [review]

**Pull request merged**